### DISCUSSION TITLE
Contribution数の内訳を見たい

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#160] [要望・アイデア・不具合]
Contribution数の再計算が実施された（ #146 ）とのことで、自分は編集リクエスト機能をよく使うこともあって結構数値が変動しているようでした。
しかしながら具体的な内訳までは分からないため、Contributionsページ（ `/:user_id/contributions` ）などでContribution数の内訳が見れると嬉しいです。

イメージとしては、GitHubのユーザーページにあるContributionのグラフのようなものなどがあるとよいです。

<img width="904" alt="スクリーンショット 2022-02-24 0 17 40" src="https://user-images.githubusercontent.com/20086673/155348776-61e1c864-9179-4565-870f-306186a79a47.png">

**Comment by kskwtnk** [comment]
@munierujp 
ご意見ありがとうございます！

たしかに仰るとおりですね……！
こちらのDiscussion https://github.com/increments/qiita-discussions/discussions/88 でも少しだけ触れていたのですが、マイページの情報自体を見直そうという話をしていまして、その一環で議論させていただきます！
